.jenkins/pytorch/win-test-helpers/test_python_first_shard.bat

```diff
@@ -14,7 +14,16 @@ if ERRORLEVEL 1 exit /b 1
 if ERRORLEVEL 1 exit /b 1
 
 echo Run nn tests
-python run_test.py --exclude-jit-executor --shard 1 2 --verbose --determine-from="%1"
+
+if "%RUN_SMOKE_TESTS_ONLY%"=="1" (
```

Comment:
I was under the impression that we do smoke test because of it is fast and contains a very small subset of tests. 

in this case, why we do we smoke test on shard? (I thought the whole point of sharding is to make sure the entire test collection run twice as fast?)

Reply:
Hm this is so it is possible to enable sharding for smoke tests too, though it's not strictly necessary. This would allow us to maintain the current CI structure as it'd run the full test suite on master (where sharding would help) and smoke tests on PRs (where sharding shouldn't hurt).

Reply:
Another alternative I can do is to ignore shards when running smoke tests and skip all the non-1 shards. That would work as well, if you think that is better?